src/main/java/cz/upce/fei/inptp/zz/entity/CourseAction.java

```diff
@@ -18,12 +18,25 @@
     private Teacher teacher;
     private List<Student> students;
     private ActionType type;
+    private int capacity;
 
     public CourseAction() {
         students = new ArrayList<>();
-                
+        capacity = Integer.MAX_VALUE;
+    }
+    
+    public boolean hasCapacity() {
```

Comment:
hasCapacity is something different from check that is inside of this method

Reply:
Solved.